HEC.FDA.Model/metrics/StudyAreaConsequencesBinned.cs

```diff
@@ -317,21 +318,14 @@ public double ConsequenceExceededWithProbabilityQ(double exceedanceProbability,
     /// <param name="damageCategory"></param>
     /// <param name="assetCategory"></param>
     /// <param name="impactAreaID"></param>
-    /// <returns></returns>
+    /// <returns>returns the existing result, else null if no result exists for that combination.</returns>
     public AggregatedConsequencesBinned GetConsequenceResult(string damageCategory, string assetCategory, int impactAreaID = utilities.IntegerGlobalConstants.DEFAULT_MISSING_VALUE)
     {
+        //Try to get specific result from the list. combination of damcat, asscat, imparea
         AggregatedConsequencesBinned result = ConsequenceResultList
             .FilterByCategories(damageCategory, assetCategory, impactAreaID)
             .FirstOrDefault();
-        if (result != null)
-        {
-            return result;
-        }
-        string message = "The requested damage category - asset category - impact area combination could not be found. An arbitrary object is being returned";
-        ErrorMessage errorMessage = new(message, MVVMFramework.Base.Enumerations.ErrorLevel.Fatal);
-        ReportMessage(this, new MessageEventArgs(errorMessage));
-        AggregatedConsequencesBinned dummyResult = new();
-        return dummyResult;
+        return result; ////TODO CHECK REFERENCES TO THIS METHOD AND MAKE THEM ADHERE TO THE NEW WORLD ORDER. ///////////9*******************************************************
```

Comment:
I see my writing again. nuke it. was just an obnoxious flag so we'd know to return here.